android/src/main/java/com/swmansion/reanimated/nodes/InvokeNode.java

```diff
@@ -0,0 +1,33 @@
+package com.swmansion.reanimated.nodes;
+
+import com.facebook.react.bridge.ReadableMap;
+import com.swmansion.reanimated.NodesManager;
+import com.swmansion.reanimated.Utils;
+import com.swmansion.reanimated.reflection.ReanimatedAccessor;
+import com.swmansion.reanimated.reflection.ReanimatedReflectionHelper;
+
+public class InvokeNode extends Node implements ConnectedNode {
+    ReanimatedAccessor mEvalHelper;
+    protected final int[] mParams;
+    private int invocationId = 0;
```

Comment:
```suggestion
    private int mInvocationId = 0;
```

and why you need this?

Reply:
74074ce